environments/echis/app-processes.yml

```diff
@@ -11,16 +11,13 @@ celery_processes:
   echis_server0:
     celery,export_download_queue,email_queue,case_import_queue:
       concurrency: 1
-      max_tasks_per_child: 5
     background_queue,case_rule_queue,analytics_queue:
       concurrency: 1
-      max_tasks_per_child: 1
     beat: {}
     celery_periodic:
       concurrency: 1
     saved_exports_queue:
       concurrency: 1
-      max_tasks_per_child: 1
```

Comment:
Just curious that in case like saved exports queue where we suddenly add lots of tasks to be run, this would never power down the worker since it would always be actively receiving something?

Reply:
```saved_exports_queue``` does zero out, so the worker would get put down then.

But your reasoning is correct that if there are always tasks in the queue, the worker won't get put down.  Our systems don't really run like that, so it shouldn't be an issue.  Also, even ```saved_exports_queue``` has no/negligible memory leaks when it takes ~18 hours on prod to run.